caffe2/proto/torch.proto

```diff
@@ -104,5 +104,5 @@ message ModelDef {
 
   // TorchScript code that this module hierarchy depends on
   // Libs will be loaded in before the main module is compiled.
-  optional LibDef libs = 9;
+  repeated LibDef libs = 9;
```

Comment:
I think this should just list the directory root where these files live. I am inclined to avoid duplication of this information because it makes it easier to hack on the file for testing purposes.

Reply:
Because this is a change to the proto version, I am apprehensive about spreading it across multiple commits.

Reply:
Makes sense. I guess we don't even really need a field in this case, since we can just consider `libs/` the root.